### PR TITLE
Feature/delete context

### PR DESCRIPTION
**Pull request by asafshen**
fixes: https://github.com/demisto/server/issues/6693

### CODE REVIEW

File: Scripts/script-DeleteConext.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: 8f947b0f-9d53-4f0b-8656-2eb164c02ffa
+  version: 14
+name: DeleteContext
+script: 'return executeCommand(''delContext'', { key: args.key, id: args.id } );'
```
Comment (aviadl) [inline Scripts/script-DeleteConext.yml]: add a param - to delete all context keys, and then delete all context keys by iterating on the context ...
Discuss with @nitzanshapira

File: Scripts/script-DeleteConext.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: 8f947b0f-9d53-4f0b-8656-2eb164c02ffa
+  version: 14
```
Comment (aviadl) [inline Scripts/script-DeleteConext.yml]: version should -1

File: Scripts/script-DeleteConext.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: 8f947b0f-9d53-4f0b-8656-2eb164c02ffa
```
Comment (aviadl) [inline Scripts/script-DeleteConext.yml]: Give a proper id so it will be easier for us to map the data - `delContext`

**Comment by asafshen** [comment]
@aviadl please review when possible

**Pull request merged**